src/main/java/com/cloudbees/plugins/deployer/DeployEvent.java

```diff
@@ -28,15 +28,14 @@
 import hudson.model.AbstractBuild;
 import hudson.model.Cause;
 
-import java.io.Serializable;
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
 
 /**
  * @author stephenc
  */
-public class DeployEvent implements Serializable {
+public class DeployEvent {
```

Comment:
Needs a weight-in from the maintainer. The class is definitely not serializable in the current implementation. But removal of the interface breaks binary compatibility.